### ISSUE TITLE
cannot fix and report negative fix count

### ISSUE THREAD

**Issue by dereuromark** [squizlabs/PHP_CodeSniffer#1621]
A fixable error, that so far usually was fine to get fixed, does not work when referenced from another ruleset.
It then errors even though the fix (debugged) seems to be fine.

It also reports

> A TOTAL OF -28 ERRORS WERE FIXED IN 1 FILE

after looping out:

>  => Fixing file: 31/1 violations remaining [made 50 passes]... ERROR in 6 secs

The code is really just
```php
        for ($i = $startIndex + 1; $i < $endIndex; $i++) {
            if ($tokens[$i]['code'] !== T_COMMENT) {
                continue;
            }

            if (!preg_match('|^\/\*\s*@\w+ (.+)|', $tokens[$i]['content'])) {
                continue;
            }

            $fix = $phpCsFile->addFixableError('Inline Doc Block comment should be using `/** ... */`', $i, 'InlineDocBlock');
            if ($fix) {
                $phpCsFile->fixer->beginChangeset();

                $comment = $tokens[$i]['content'];
                $comment = str_replace('/*', '/**', $comment);

                $phpCsFile->fixer->replaceToken($i, $comment);

                $phpCsFile->fixer->endChangeset();
            }
        }
```

**Comment by jrfnl** [comment]
Sounds like a fixer conflict. Could you run it with `-vvv` ? The output will be long, but the relevant bit should be at the bottom and will show you which fixes PHPCS tries to make and should tell you which sniff this sniff is conflicting with.

**Comment by dereuromark** [comment]
There is indeed conflict with some other. This is rather difficult to isolate, though.

**Comment by gsherwood** [comment]
@dereuromark I know you've closed this, but if you are able to provide your custom standard or ruleset, I could take a look at where the conflict is.

The best place to start is actually with PHPCS, using the diff report and the `-vvv` CLI arg. This will print information about each fix being applied, what the file looks like after each loop, and any possible conflicts the fixer found.

e.g., phpcs --report=diff -vvv --standard=... /path/to/file.php
